react/features/base/participants/functions.js

```diff
@@ -198,8 +197,7 @@ export function getParticipantCountWithFake(stateful: Object | Function) {
 /**
  * Returns participant's display name.
  *
- * FIXME: Remove the hardcoded strings once interfaceConfig is stored in redux
- * and merge with a similarly named method in {@code conference.js}.
+ * And merge with a similarly named method in {@code conference.js}.
```

Comment:
Can you reword this in a way it makes sense? The unification with conference.js is still missing. In fact, maybe check if we can do it now?